docs/UpgradeGuide.md

```diff
@@ -0,0 +1,568 @@
+
+### Removed `Datadog.tracer`
+
+Many of the functions accessed directly through `Datadog.tracer` have been moved to `Datadog::Tracing` instead.
+
+### Context
+
+Direct usage of `Datadog::Context` has been removed. Previously, it was used to modify or access active trace state. Most of these use cases have been replaced by `TraceOperation` and have been given new APIs.
+
+## Full list of breaking changes
+
+| **Category**  | **Type** | **Description**                                                                                                      | **Change / Alternative**                                                                                                                                                  |
+|---------------|----------|----------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| General       | Changed  | Many constants have been moved from `Datadog` to `Datadog::Core`, `Datadog::Tracing`, `Datadog::Profiling`           | Update your references to these new namespaces where appropriate.                                                                                                         |
+| General       | Removed  | `Datadog.tracer`                                                                                                     | Use methods in `Datadog::Tracing` instead.                                                                                                                                |
+| General       | Removed  | Support for trace agent API v0.2.                                                                                    | Use v0.4 instead (default behavior)                                                                                                                                       |
+| CI API        | Changed  | `DD_TRACE_CI_MODE_ENABLED` environment variable is now `DD_TRACE_CI_ENABLED`.                                        | Use `DD_TRACE_CI_ENABLED` instead.                                                                                                                                        |
+| Configuration | Changed  | `Datadog.configuration` raises errors if you attempt to access non-global settings.                                  | Use `Datadog::Tracing.configuration`, `Datadog::Profiling.configuration`, or `Datadog::CI.configuration` when appropriate.                                                |
+| Configuration | Changed  | `Datadog.configure` raises errors if you attempt to configure non-global settings.                                   | Use `Datadog::Tracing.configure`, `Datadog::Profiling.configure`, or `Datadog::CI.configure` when appropriate.                                                            |
```

Comment:
Is this still a thing after last week's changes?

Reply:
No, it isn't. Will remove this.